todo_app/templates/index.html

```diff
@@ -13,7 +13,7 @@ <h2>Items</h2>
       <ul class="list-group mb-4">
         {% for item in items %}
         <li>
-          Id: {{item.id}} - {{ item.title }} - {{item.status}}
+          Id: {{item.id}} - {{ item.name }} - {{item.status}}
         </li>
```

Comment:
In addition to viewing and creating cards, the exercise asks you to add a new feature: changing the status of cards.

Do this by adding a new form here and a corresponding route in app.py that ultimately sends a PUT request to the Trello API's /cards/card-id endpoint to update it. Change the value of the card's "idList" in order to change its status from "To do" to "Done"